src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendarPadding;
+  gap: 1px;
+  width: calendarWidth();
```

Comment:
Use function when you need to set some parameters during the call, in this case you could create just a new variable and calc everything there